it/uk/gov/hmrc/fhregistrationfrontend/controllers/BusinessPartnerControllerIntegrationSpec.scala

```diff
@@ -0,0 +1,57 @@
+package uk.gov.hmrc.fhregistrationfrontend.controllers
+
+import org.jsoup.Jsoup
+import play.api.libs.ws.DefaultWSCookie
+import play.api.test.WsTestClient
+import uk.gov.hmrc.fhregistrationfrontend.testsupport.{Specifications, TestConfiguration}
+
+class BusinessPartnerControllerIntegrationSpec
+  extends Specifications with TestConfiguration {
+
+
+  "GET /form/business-partner" when {
+
+    "render the business partners page" when {
+      "the user is authenticated" in {
+        given.commonPrecondition
+
+        WsTestClient.withClient { client =>
+          val result = client.url(s"$baseUrl/business-partners")
+            .addCookies(DefaultWSCookie("mdtp", authAndSessionCookie)).get()
+
+          whenReady(result) { res =>
+            res.status mustBe 200
+            val page = Jsoup.parse(res.body)
+            page.title must include("Business partners - Apply for the Fulfilment House Due Diligence Scheme")
+          }
+        }
+      }
+    }
+
+  }
+
+  "POST /business-partners" when {
```

Comment:
Could add more tests for error message scenario. 

Also would be best to have a test for submitting for each option. Ensuring the correct information is submitted each time.